### PR TITLE
[TASK] Use fluid to render reports

### PR DESCRIPTION
**Pull request by georgringer**
Using fluid instead of a custom template solution improves the code quality
and makes later changes easier.
Furthermore the output has been improved by using proper tags like <code> for paths and versions</code>

Resolves: #739

### CODE REVIEW

File: Classes/Report/AccessFilterPluginInstalledStatus.php [hunk 1]
```diff
@@ -63,7 +65,7 @@ class AccessFilterPluginInstalledStatus implements StatusProviderInterface
     public function getStatus()
     {
         $reports = array();
-        $solrConnections = GeneralUtility::makeInstance('ApacheSolrForTypo3\\Solr\\ConnectionManager')->getAllConnections();
+        $solrConnections = GeneralUtility::makeInstance(ConnectionManager::class)->getAllConnections();
```
Comment (NamelessCoder) [inline Classes/Report/AccessFilterPluginInstalledStatus.php]: This and other `::class` should probably not be part of the PR - although I agree they should be used!

File: Classes/Report/AccessFilterPluginInstalledStatus.php [hunk 2]
```diff
@@ -94,27 +96,19 @@ protected function checkPluginInstallationStatus(SolrService $solrConnection
...
-                . '<li>Host: ' . $solrConnection->getHost() . '</li>'
-                . '<li>Port: ' . $solrConnection->getPort() . '</li>'
-                . '<li>Path: ' . $solrConnection->getPath() . '</li>
-				</ul>';
-
-            $message .= $this->getPluginDownloadMessage();
-
-            $status = GeneralUtility::makeInstance('TYPO3\\CMS\\Reports\\Status',
+            $standaloneView = GeneralUtility::makeInstance(StandaloneView::class);
+            $standaloneView->setTemplatePathAndFilename(
```
Comment (NamelessCoder) [inline Classes/Report/AccessFilterPluginInstalledStatus.php]: You could give this one a (set of) templateRootPath and then call `render($virtualAction)` - that would allow integrators to replace the template. Same for the case below :)

File: Classes/Report/AccessFilterPluginInstalledStatus.php (hunk 1 above)
Comment (timohund) [inline Classes/Report/AccessFilterPluginInstalledStatus.php]: I would say both is ok, since we could change it when we work on the code anyway. Doing this in multiple pull requests is better for sure, but i think we could also be pracmatically and change it when we work in these parts.

File: Classes/Report/AccessFilterPluginInstalledStatus.php (hunk 2 above)
Comment (timohund) [inline Classes/Report/AccessFilterPluginInstalledStatus.php]: Thanks for the hint. I would merge it, since the pull requests does a lot improvements and we can add this later.
